botocore/docs/utils.py

```diff
@@ -124,3 +125,49 @@ def document_auto_populated_param(self, event_name, section, **kwargs):
             section = section.get_section('structure-value')
             if self.name in section.available_sections:
                 section.delete_section(self.name)
+
+
+class HideParamFromOperations(object):
+    """Hides a single parameter from multiple operations.
+
+    This method will remove a parameter from documentation and from
+    examples. This method is typically used for things that are
+    automatically populated because a user would be unable to provide
+    a value (e.g., a checksum of a serialized XML request body)."""
+    def __init__(self, service, parameter_name, operation_names):
```

Comment:
I would prefer to stay consistent and change ``service`` to ``service_name`` in order to help remove ambiguity. Would help to have it included in the doc string for it as well.